docs/public/replay-viewer/style.css

```
@@ -0,0 +1,565 @@
+  opacity: 0;
+  transition: opacity 160ms ease;
+}
+
+.player-chrome[data-visible="true"] {
+  opacity: 1;
+}
+
+.player-chrome::before {
+  content: "";
+  position: absolute;
+  inset: auto 0 0 0;
```

Stylelint (17.11.0)

[error] Expected "auto 0 0 0" to be "auto 0 0" (shorthand-property-no-redundant-values)

(shorthand-property-no-redundant-values)